Documentation/concepts/ebpf/lifeofapacket.rst

```diff
@@ -8,6 +8,15 @@
 Life of a Packet
 ################
 
+This documentation page overviews the life of a packet from an eBPF datapath perspective 
+by explaining three different scenarios. 
+
+.. admonition:: Video
+  :class: attention
+
+  You can also watch a video explanation of this page that also explores the tools available within Cilium to help 
```

Comment:
```suggestion
  You can also watch a video explanation of this topic that also explores the tools available within Cilium to help 
```
(The video does not focus on “this page” :slightly_smiling_face:)

Reply:
Thank you.